<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="PlotWidget" name="gLeakage"/>
      </item>
      <item>
       <widget class="PlotWidget" name="gOscilloscope"/>
      </item>
      <item>
       <widget class="PlotWidget" name="gValueVsV"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Gate Start (V):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tGateStart"/>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Gate Step (V):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tGateStep"/>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Gate End (V):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tGateEnd"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Int Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tIntStart">
            <property name="text">
             <string>7e-6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Int End</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tIntEnd">
            <property name="text">
             <string>20e-6</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Measure Every</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="tMeasureEvery">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="bStartScan">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bAbortScan">
          <property name="text">
           <string>Abort</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Osc Channel:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cChannelPicker">
        <item>
         <property name="text">
          <string>CH1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CH2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CH3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CH4</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Save name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tSave"/>
      </item>
      <item>
       <widget class="QPushButton" name="bSaveDirectory">
        <property name="text">
         <string>Choose Location</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bResetLeak">
        <property name="text">
         <string>Reset Leak</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="mFileUpdate"/>
    <addaction name="mFileExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mFileExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="mFileUpdate">
   <property name="text">
    <string>Update Scope</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
